packages/engine/src/avatar/functions/moveAvatar.ts

```diff
@@ -117,7 +119,7 @@ export function updateLocalAvatarPosition(additionalMovement?: Vector3) {
     const hit = groundHits[0]
     const controllerOffset = controller.controller.offset()
     // controller.isInAir = !grounded
-    controller.isInAir = hit.distance > 1 + controllerOffset * 1.5
+    controller.isInAir = hit.distance > 1 + controllerOffset * 10
```

Comment:
I cranked the controller offset multiplier value up to 10 experimentally. This prevents the value from erroneously returning true on the sky station stairs, causing the avatar to immediately fall down when using autopilot. In my testing I did not find any issues caused by this higher value.

Reply:
Please check on puttclub, this will likely cause unintended consequences

Reply:
I tested location/puttclub/alpha on desktop by running/jumping around and noticed no major difference, actually an improvement I saw by setting the value to 10 was that isInAir is less frequently erroneously set to true when moving between minor obstacles, like the edges of golf courses. This means no sudden transition to mid air animations in my testing. That said, a common issue in both cases is that the avatar would get stuck on tiny barriers, when you would expect a sliding motion.

Any other specific testing you have in mind?

Reply:
In fact, there are still some odd edge cases in which isInAir is set to true when it shouldn't be. Because autopilot has to return if we are no longer grounded, this can obviously cause some issues.